R/derive_param_tte.R

```diff
@@ -845,8 +889,10 @@ extend_source_datasets <- function(source_datasets,
 #'
 #' @param order Sort order
 #'
-#'   If the argument is set to a non-null value, for each by group the first or
-#'   last observation
+#'   An optional named list returned by `exprs()` defining additional variables
+#'   that the input dataset is sorted on after `date`.
```

Comment:
```suggestion
#'   that the source dataset is sorted on after `date`.
```
Stefan used "source" and it is more in keeping with the argument naming, rather than my suggested "input"